addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/handler/YamahaBridgeHandler.java

```diff
@@ -0,0 +1,283 @@
+
+            updateProperty(YamahaReceiverBindingConstants.PROPERTY_VERSION, state.version);
+            updateProperty(YamahaReceiverBindingConstants.PROPERTY_ASSIGNED_NAME, state.name);
+
+        } catch (IOException e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+            state.invalidate();
+            return;
+        } catch (ParserConfigurationException e) {
+            e.printStackTrace();
+        } catch (SAXException e) {
+            e.printStackTrace();
+        }
+
+        Bridge bridge = (Bridge) thing;
+        List<Thing> things = bridge.getThings();
+        for (Thing thing : things) {
+            YamahaZoneThingHandler handler = (YamahaZoneThingHandler) thing.getHandler();
+            // If thing still thinks that the bridge is offline, update its status.
+            if (thing.getStatusInfo().getStatusDetail() == ThingStatusDetail.BRIDGE_OFFLINE) {
+                handler.bridgeStatusChanged(ThingStatusInfoBuilder.create(bridge.getStatus()).build());
+            } else {
+                handler.updateZoneInformation();
+            }
+        }
+        updateStatus(ThingStatus.ONLINE);
+    }
+
+    @Override
+    public void handleConfigurationUpdate(Map<String, Object> configurationParameters) {
+        if (!isInitialized()) {
+            super.handleConfigurationUpdate(configurationParameters);
+            return;
+        }
+
+        validateConfigurationParameters(configurationParameters);
+
+        // can be overridden by subclasses
+        Configuration configuration = editConfiguration();
+        for (Entry<String, Object> configurationParmeter : configurationParameters.entrySet()) {
+            configuration.put(configurationParmeter.getKey(), configurationParmeter.getValue());
+        }
+
+        updateConfiguration(configuration);
+
+        // Check if host configuration has changed
+        String host_config = (String) thing.getConfiguration().get(YamahaReceiverBindingConstants.CONFIG_HOST_NAME);
+        if (host_config != null) {
+            xml.setHost(host_config);
+        }
+
+        // Check if refresh configuration has changed
+        BigDecimal interval_config = (BigDecimal) thing.getConfiguration()
+                .get(YamahaReceiverBindingConstants.CONFIG_REFRESH);
+        if (interval_config != null && interval_config.intValue() != refrehInterval) {
+            setupRefreshTimer(interval_config.intValue());
+        }
+
+        // Read the configuration for the relative volume change factor.
+        BigDecimal relVolumeChange = (BigDecimal) thing.getConfiguration()
+                .get(YamahaReceiverBindingConstants.CONFIG_RELVOLUMECHANGE);
+        if (relVolumeChange != null) {
+            relativeVolumeChangeFactor = relVolumeChange.floatValue();
+        } else {
+            relativeVolumeChangeFactor = 0.5f;
+        }
+
+        Boolean configTraceLog = (Boolean) thing.getConfiguration()
```

Comment:
I haven't seen an explicit hint regarding additional log files. Especially for long trace output I would prefer to write an own log file instead of filling-up the standard log.